src/ResourceManager/Network/Commands.Network/Commands.Network.csproj

```diff
@@ -57,11 +79,20 @@
       <HintPath>..\..\..\packages\Microsoft.Azure.KeyVault.Core.1.0.0\lib\net40\Microsoft.Azure.KeyVault.Core.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Microsoft.Azure.Management.Network, Version=15.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.15.1.0-preview\lib\net452\Microsoft.Azure.Management.Network.dll</HintPath>
-      <Private>True</Private>
+    <Reference Include="Microsoft.Azure.Management.Network, Version=19.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=AMD64">
```

Comment:
Remove the redundant references

Reply:
so there's a discrepancy here :
```
    <Reference Include="Microsoft.Azure.Management.Network, Version=19.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=AMD64"> 
    <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.21.1.0-preview\lib\net452\Microsoft.Azure.Management.Network.dll</HintPath>  
      <Private>True</Private> 
    </Reference>
```
can you make the version number same in both the places?